pkg/sql/logictest/testdata/logic_test/create_statements

```diff
@@ -54,16 +54,16 @@ CREATE TABLE crdb_internal.backward_dependencies (
    dependson_details STRING NULL
 )  {}  {}
 CREATE TABLE crdb_internal.builtin_functions (
-   function STRING NOT NULL,
-   signature STRING NOT NULL,
-   category STRING NOT NULL,
-   details STRING NOT NULL
-)  CREATE TABLE crdb_internal.builtin_functions (
-   function STRING NOT NULL,
-   signature STRING NOT NULL,
-   category STRING NOT NULL,
-   details STRING NOT NULL
-)  {}  {}
+                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          function STRING NOT NULL,
```

Comment:
i think these extra lines of whitespace were added accidentally

Reply:
Fixed.